editor/src/components/text-editor/text-editor.tsx

```diff
@@ -357,6 +359,24 @@ const TextEditor = React.memo((props: TextEditorProps) => {
     myElement.current.textContent = firstTextProp
   }, [firstTextProp])
 
+  React.useEffect(() => {
+    if (myElement.current == null) {
```

Comment:
It seems this should live in the `useEffect` below, since that is also checking `myElement.current == null` and just early returning if true

Reply:
Actually, on second thought I think you're right to separate this and provide no dependencies.

Reply:
Great suggestion, added it https://github.com/concrete-utopia/utopia/pull/3893/commits/c053a6cdf30cfbdd793e188a5c17d30e9412bc0c